pkg/services/ngalert/api/api_ruler_base.go

```diff
@@ -33,12 +33,12 @@ type RulerApiBase struct {
 
 func (api *API) RegisterRulerApiEndpoints(srv RulerApiService) {
 	api.RouteRegister.Group("", func(group routing.RouteRegister) {
```

Comment:
Is there some reason we can't make the the first argument to the `Group` method `"/api/ruler"` in this and other sections to shorten the repetition?

Reply:
good point! This is auto-generate from [this](https://github.com/grafana/grafana/blob/664268a49803edfbd0cd25378e39ae6667901644/pkg/services/ngalert/api/swagger-codegen/templates/controller-api.mustache#L25) template so I hadn't found an easy way to make it work for all the routes.

Reply:
Oh auto-gen, doesn't matter then :-)